Gemfile

```diff
@@ -17,6 +17,8 @@ else
   gem 'journey'
 end
 
+gem 'i18n', '~> 0.6.11'
```

Comment:
It does work, but Ruby 1.8, and 1.9.2 test suite fails if we don't specify the version.